start-workflow-from-archive/StartWorkflow.py

```diff
@@ -0,0 +1,106 @@
+import argparse
+import requests
+import sys
+from requests.auth import HTTPDigestAuth
+
+SERVER_URL = "https://stable.opencast.org"
+DIGEST_USER_NAME = "opencast_system_account"
+DIGEST_PASSWORD = "CHANGE_ME"
+
+
+class OpencastException(BaseException):
+
+    def __init__(self, message):
+        self.message = message
+
+
+def start_workflow(mediaPackageId: str, workflowId: str, workflowParams: list = None) -> object:
```

Comment:
Weird function annotation. Function annotations can only be used with Python 3 and are hence still quite uncommon. But setting `object` as the expression is weird in any case. Just omit them as they are completely optional or take a look at [PEP 3107](https://www.python.org/dev/peps/pep-3107/) for details.